src/client/ui/select-element.js

```diff
@@ -36,6 +37,23 @@ function onDocumentMouseDown (e) {
         collapseOptionList();
 }
 
+function onWindowClick (e, dispatched, preventDefault) {
+    const target   = e.target || e.srcElement;
+    const optionIndex = arrayUtils.indexOf(options, target);
+
+    if (optionIndex < 0)
+        return;
+
+    preventDefault();
+
+    const isDisabled = target.className.indexOf(DISABLED_CLASS) > -1;
```

Comment:
can we just check here `target.isDisabled`?

Reply:
no, it does not work in IE. Probably it is an other bug